media/css/firefox/private_browsing/private-browsing.less

```diff
@@ -0,0 +1,489 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+/*-------------------------------------------------------------------------*/
+// Common
+
+#wrapper {
+    border-top: 2px solid #fff;
+    background-color: #0095dd;
```

Comment:
Should this be @mozIDBlue https://github.com/mozilla/bedrock/blob/master/media/css/sandstone/lib.less#L66